src/modules/pokemons/PokemonList.ts

```diff
@@ -8316,7 +8333,7 @@ export const pokemonList = createPokemonArray(
         'levelType': LevelType.mediumfast,
         'exp': 175,
         'eggCycles': 25,
-        // 'evolutions': [MegaEvolution('Scizor', 'Mega Scizor')],
+        //'evolutions': [MegaEvolution(MegaStoneType.Scizorite, 'Scizor', 'Mega Scizor')],
```

Comment:
```suggestion
        'evolutions': [MegaEvolution(MegaStoneType.Scizorite, 'Scizor', 'Mega Scizor')],
```

Reply:
This is intentionally commented out in case we don't release the Friend Safari next update. It would be resolved in the "opening the Friend Safari" PR.